packages/@o3r/store-sync/src/sync-storage/storage-sync.ts

```diff
@@ -74,9 +74,12 @@ export const rehydrateApplicationState = (
     let reviver = restoreDates ? dateReviver : dummyReviver;
     let deserialize: ((arg0: string) => any) | undefined;
     let decrypt: ((arg0: string) => string) | undefined;
+    let syncForFeature = false;
```

Comment:
You removed your previous `forFeature` option at `StorageConfig` and replace it by a static default value `false`.
Is it because having a default value different from `false` does not make sense?

Reply:
My understanding is that at the moment, every DES fork including RefX is syncing the stores at the root level and if I keep the default as true and they start using this method to sync the store instead of the ngrx-store-localstorage, they will encounter issues. So, basically to maintain backward compatibility, I prefer to keep the default as false.

Reply:
So, essentially the practice has been to register the meta reducers at the root level in the app module. So, keeping the default as true would not work. So, to make it backwards compatible I believe we can keep the default as false.